pkg/preparer/orchestrate.go

```diff
@@ -418,7 +420,9 @@ func (p *Preparer) installAndLaunchPod(pair ManifestPair, pod Pod, logger loggin
 			}
 		} else {
 			// TODO: do this in a transaction
-			err = p.podStore.WriteRealityIndex(pair.PodUniqueKey, p.node)
+			// TODO dai - what do I do with the errors?
```

Comment:
as a general statement i think errors should always be checked. It's one of the interesting things about go, it forces you to think through all of these cases.

I think we should still just keep looping trying to see if we get a different result from `Add()`. If we ended up in the position where that was failing, we'd hopefully know pretty quickly by looking at error rate graphs in the preparer (which count the number of Error() log lines) or by customer reports